requirements-production.txt

```diff
@@ -19,7 +19,7 @@ cssutils==2.11.1
 html2text==2020.1.16
 tablib==3.5.0
 django-redis>=5.2.0,<5.3
-django-select2>=6.3.1,<6.4
+django-select2>=6.3.1,<8.3
```

Comment:
looks like `7.3.1` isnt available for pip. Ill take the next one over (`7.4.1`)
```
(bytedeck) D:\#\BYTEDECK_FORK\bytedeck\src>pip index versions django-select2
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
django-select2 (8.2.1)
Available versions: 8.2.1, 8.2.0, 8.1.2, 8.1.1, 8.1.0, 8.0.0, 7.11.1, 7.11.0, 7.10.1, 7.10.0, 7.9.0, 7.8.1, 7.7.3, 7.7.2, 7.7.1, 7.7.0, 7.6.2, 7.6.1, 7.6.0, 7.5.0, 7.4.2, 7.4.1, 7.3.0, 7.2.4, 7.2.3, 7.2.2, 7.2.0, 7.1.2, 7.1.1, 7.1.0, 7.0.5, 7.0.4, 7.0.3, 7.0.2, 6.3.1, 6.3.0, 6.2.0, 6.1.2, 6.1.1, 6.1.0, 6.0.3, 6.0.2, 6.0.1, 6.0.0, 5.11.1, 5.11.0, 5.10.0, 5.9.0, 5.8.10, 5.8.9, 5.8.7, 5.8.6, 5.8.5, 5.8.4, 5.8.3, 5.8.2, 5.8.1, 5.8.0, 5.7.1, 5.7.0, 5.6.0, 5.5.0, 5.4.3, 5.4.2, 5.4.1, 5.4.0, 5.3.1, 5.3.0, 5.2.1, 5.2.0, 5.1.1, 5.1.0, 5.0.3, 5.0.2, 5.0.1, 4.3.2, 4.3.1, 4.3.0, 4.2.2, 4.2.1, 4.2.0, 4.0.0, 3.3.1, 3.3.0, 3.2.1, 3.2.0, 3.1.5, 3.1.4, 3.1.3, 3.1.2, 3.1.1, 3.1.0, 3.0.2, 3.0.1, 3.0, 2.0.1
  INSTALLED: 6.3.1
  LATEST:    8.2.1
```